### PR TITLE
reverso 2.3.0

### PR DESCRIPTION
**Pull request by alexreg**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by alexreg** [comment]
I presume the download URL used in the cask is preferable to the CDN one, i.e., `https://cdn.reverso.net/download/reverso/desktop/macos/Reverso.dmg`.

**Comment by bevanjkay** [comment]
@alexreg Ideally, we would use the url from within the appcast `https://cdn.reverso.net/download/reverso/desktop/macos/distrib/Reverso_2.3.0.317.zip` so that we can add a shasum.

File: Casks/reverso.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+cask "reverso" do
+  version "2.3.0"
+  sha256 :no_check
+
+  url "https://dl.reverso.net/desktop-app/macos"
```
Comment (bevanjkay) [inline Casks/reverso.rb:5]:
```suggestion
  url "https://cdn.reverso.net/download/reverso/desktop/macos/distrib/Reverso_#{version.csv.first}.#{version.csv.second}.zip"

File: Casks/reverso.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+cask "reverso" do
+  version "2.3.0"
```
Comment (bevanjkay) [inline Casks/reverso.rb:2]:
```suggestion
  version "2.3.0,317"
```

File: Casks/reverso.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  sha256 :no_check
```
Comment (bevanjkay) [inline Casks/reverso.rb:3]:
```suggestion
  sha256 "28ed063970d5f74135107403cb183bab27d212c6e63ef732692ad20c4fac6c15"
```

File: Casks/reverso.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+  url "https://dl.reverso.net/desktop-app/macos"
+  name "Reverso"
+  desc "Translate text from any app or website in just one click"
+  homepage "https://context.reverso.net/translation/windows-mac-app"
+
+  livecheck do
+    url "https://cdn.reverso.net/download/reverso/desktop/macos/appcast.xml"
+    strategy :sparkle, &:short_version
```
Comment (bevanjkay) [inline Casks/reverso.rb:12]:
```suggestion
    strategy :sparkle
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]
Thanks @alexreg - these changes will allow the cask to use a shasum.

**Comment by alexreg** [comment]
@bevanjkay That makes sense. Thanks for the review – changes integrated.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**